src/components/Select/Select.tsx

```diff
@@ -124,7 +124,6 @@ const Select = React.forwardRef<HTMLInputElement, Props & InputProps & TestProps
     } = useMultiselectUtils({
       selectedOptions,
       options,
-      setOpen,
```

Comment:
do you need to remove this and keep your state? Can we have this logic on the hook?

Reply:
this state is needed in the Select component (so it cannot stay only in the scope of the hook) and besides that, the `useMultiselectUtils` hook holds all the necessary logic for the MultiSelect field. the `[open, setOpen]` state applies to all Select field types so semantically speaking it should be on the Select component level